go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/screwdriver-cd/launcher
+
+go 1.12
+
+require (
+	github.com/mattn/go-colorable v0.1.2 // indirect
+	github.com/mattn/go-isatty v0.0.9 // indirect
+	github.com/myesui/uuid v1.0.0 // indirect
+	github.com/peterbourgon/mergemap v0.0.0-20130613134717-e21c03b7a721
+	gopkg.in/fatih/color.v1 v1.7.0
+	gopkg.in/kr/pty.v1 v1.1.8
```

Comment:
the https://github.com/kr/pty repo says: "New clients should use the new module import path github.com/creack/pty", should we migrate to it?

Reply:
Done